stdlib/testing/testing.flux

```diff
@@ -11,14 +11,14 @@ option loadMem = (csv) => c.from(csv: csv)
 
 inspect = (case) => {
     tc = case()
-    got = tc.input |> tc.fn() |> yield(name: "_test_result")
-    dif = got |> diff(want: tc.want) |> yield(name: "diff")
+    got = tc.input |> tc.fn()
+    dif = got |> diff(want: tc.want)
     return {
         fn:    tc.fn,
         input: tc.input
         want:  tc.want,
-        got:   got,
-        diff:  dif,
+        got:   got |> yield(name: "_test_result"),
+        diff:  dif |> yield(name: "diff"),
```

Comment:
This change looks like a noop. Why does moving the yields down to these lines change anything?

Reply:
`_test_result` is no longer returned as part of dif. Only the diff tables are returned.

Reply:
ah that makes sense